### ISSUE TITLE
Traffic Options

### ISSUE THREAD

**Issue by ShaylinChetty** [WiredSolutions/react-azure-maps#60]
Traffic options don't seem to be passing through. Unless I am misunderstanding something it is expected that the following map would show traffic events. 

```typescript
<>
    <AzureMapsProvider>
        <AzureMap options={option} trafficOptions={{ incidents: true, flow: "relative" }}>
        </AzureMap>
    </AzureMapsProvider>
</>
```

Traffic options seem unused in the playground examples.

**Comment by srednicki95** [comment]
Thanks for your feedback, we will investigate on that and add sample of traffic options to playgroud.

**Comment by srednicki95** [comment]
This issue has been fixed in 0.1.2 version. Playground example will be updated soon.
